app/javascript/components/CalendarSubmission.js

```diff
@@ -20,7 +21,7 @@ const CalendarSubmission = props => {
             animated
             now={66}
             variant="success"
-            label="Step 2/3"
+            label="Step 2/4"
```

Comment:
I personally think vendor selection should be step 2/4 and then selecting a final time should be step 3/4 and then selecting the calendar you want to should be step 4/4. Not sure how easy the code is to do that though.